File: ruby/hello.rb
require_relative 'bundle/bundler/setup'
require 'json'
require 'iron_mq'
require 'iron_worker'

puts "Hello #{IronWorker.payload["name"]}!"
puts "Here is the payload: #{IronWorker.payload}"
puts "Here is my special env variable: #{ENV['YOUR_ENV_VAR']}"

File: python/hello.py
import sys
sys.path.append("packages")
import iron_mq
from iron_worker import *

print 'Hello %s' %  IronWorker.payload()['name']
print
print 'Here is the payload: %s' %  IronWorker.payload()
print 'Here is the task_id: %s' %  IronWorker.task_id()
